src/cli/reinstall.rs

```diff
@@ -0,0 +1,112 @@
+use crate::cli::cli_config::WorkspaceConfig;
+use crate::environment::get_update_lock_file_and_prefix;
+use crate::lock_file::UpdateMode;
+use crate::{UpdateLockFileOptions, WorkspaceLocator};
+use clap::Parser;
+use fancy_display::FancyDisplay;
+use itertools::Itertools;
+use pixi_config::ConfigCli;
+
+/// Re-install an environment, both updating the lockfile and re-installing the environment.
+///
+/// This command reinstalls an environment, if the lockfile is not up-to-date it will be updated.
+/// If `--package` is given, only the specified package will be reinstalled.
```

Comment:
I would say we want to make this without a flag? Like how `pixi add` works. This is more user-friendly in my opinion.

Reply:
Yes, I think you are right. Good point!